yt_dlp/extractor/gab.py

```diff
@@ -0,0 +1,64 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    clean_html,
+    str_to_int,
+)
+
+
+class GabTVIE(InfoExtractor):
+    _VALID_URL = r'(?:https?://)tv.gab.com/channel/[^/]+/view/(?P<id>[^/]+)'
+    _TESTS = [{
+        'url': 'https://tv.gab.com/channel/wurzelroot/view/why-was-america-in-afghanistan-61217eacea5665de450d0488',
+        'info_dict': {
+            'id': '61217eacea5665de450d0488',
+            'ext': 'mp4',
+            'title': 'WHY WAS AMERICA IN AFGHANISTAN - AMERICA FIRST AGAINST AMERICAN OLIGARCHY',
+            'description': None,
+            'uploader': 'Wurzelroot',
+            'uploader_id': '608fb0a85738fd1974984f7d',
+            'thumbnail': 'https://tv.gab.com/image/61217eacea5665de450d0488',
+        }
+    }]
+
+    def _real_extract(self, url):
+        id = self._match_id(url).split('-')[-1]
```

Comment:
it's better if `match_id` can get the id directly

Reply:
this is more stable imo